### ISSUE TITLE
Build a  sample application to run OpenCV on KubeEdge platform

### ISSUE THREAD

**Issue by dingyin** [kubeedge/examples#39]
OpenCV (Open source computer vision) is a library of programming functions mainly aimed at real-time computer vision. OpenCV runs on the mobile OS too.

To demonstrate KubeEdge platform, we need to build a  sample application to run OpenCV on KubeEdge platform, ideally running inference on the edge and training on the cloud, showing cloud-edge collaboration.

**Comment by christyjacob4** [comment]
@dingyin I'd like to work on this as part of GSOC 2020
I'm thinking of implementing a website like https://teachablemachine.withgoogle.com/train that will enable us to train on the cloud and run inference on the edge. 

I've worked with Deep learning and computer vision projects before and you can have a look at this https://handwriting-recognition.netlify.com/#/

**Comment by dingyin** [comment]
@christyjacob4 Cool. Can you come up with a proposal with more details?

**Comment by christyjacob4** [comment]
@dingyin  Yes i'm working on it. Can you tell me where I can reach out for help? Is the KubeEdge Slack channel the right place??

**Comment by Urhengulas** [comment]
Hello @dingyin 
I would also love to work on this as part of GSOC2020. Should I send you my proposal before the deadline or post it here?

**Comment by dingyin** [comment]
@Urhengulas Great. We set up a new channel #summer-of-code in KubeEdge Slack. You are welcome to join the channel to discuss the proposal etc.

**Comment by dingyin** [comment]
@christyjacob4 @Urhengulas Do you get your proposal ready?

**Comment by astoycos** [comment]
@dingyin I would love to help out with this as well, More specifically I am interested in the Live Edge Video Streaming/Analytics use case where as you suggested, analytics would occur on the edge node and general training would revert back to the cloud.  Do you think that use case would be in the scope of this issue?

**Comment by ijelliti** [comment]
@dingyin I would love work on this project. I have used OpenCV on many data-science projects before

**Comment by dingyin** [comment]
@astoycos @ijelliti Great. Are you guys in KubeEdge Slack? We can chat more details.

**Comment by ijelliti** [comment]
@dingyin yes I am. Any specific slack channel for this topic.

**Comment by astoycos** [comment]
I am as well @dingyin

**Comment by daixiang0** [comment]
@dingyin @ibrahimjelliti @astoycos is it still in process?

**Comment by ibrahimjelliti** [comment]
@daixiang0 for me its not in process

**Comment by a732264856** [comment]
I am very interested in knowing your question. Where can I find more detailed information?
